### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by ogunkakisim** [Chia-Network/chia-blockchain#8169] [Support]
Hello, I want to use Raspberry PI 4 and Raspberry OS 64-Bit for Chia Farm, but; HDDs go to sleep after a certain period of time while farming, is there a way to prevent this? All disks are WD External Mybook, my USB hub is TP Link Uh 720. Thanks

- PC: Raspberry PI 4 4GB RAM
- SD: Sandisk Extreme Pro 64GB
- Ext HDD's. Wd Mybook
- OS: Raspberry OS
- Hub: TP Link Uh-720
- CPU: ARM Cortex 64

**Comment by BrandtH22** [comment]
@ogunkakisim , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - this is an expected event that occurs with OS's that are set to automatically spin down drives when not active. Community members have built scripts to keep drives alive and some OSs have settings to this as well. Let us know if you would like assistance in getting this setup.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
